src/main/java/core/basesyntax/Diamond.java

```diff
@@ -6,13 +6,40 @@
 
     /**
      * Нарисовать ромб со звёздочек заданной величины.
-     *
      * Пример: величина 3, значит максимальное кол-во звёзд в ряду (по диагонали) будет 3.
-     *
      * Вместо System.out.print() или System.out.println() использовать stream.print()
      * или stream.println()
      */
     public static void drawDiamond(int maxSize, PrintStream stream) {
+        int i = 1;
```

Comment:
can we simplify this method?

Reply:
we can split into two cycles. then the method will be simpler